floodviz/static/js/hydrograph.js

```diff
@@ -6,6 +6,7 @@
  *        @prop 'width' v(int) - width of the graph
  *        @prop 'div_id' v(string) - id for the container for this graph
  *        @prop 'display_ids' v(list) - default series to show on hydrograph
+ *        @prop 'disableInteractions' v(boolean) - disables interactions when graphic is embeddable
```

Comment:
You are mixing how wer are using this with describing the parameter. You do not need to add the "when graphic is embeddable". There may be other reasons we want to disableInteractions (like for the thumbnail generation). No need to change now.